apps/meteor/app/utils/client/lib/RestApiClient.ts

```diff
@@ -32,7 +32,7 @@ APIClient.use(async function (request, next) {
 	try {
 		return await next(...request);
 	} catch (error) {
-		if (!(error instanceof Response) || error.status !== 400) {
+		if (!(error instanceof Response)) {
```

Comment:
Wouldn't this cause problems with 401s? If the error is swallowed, I think there would be no automatic logout on 401 👀

Reply:
Any suggestions on how to test this?

Reply:
Based on this code, it seems like 401 errors will also result in an error being thrown since the promise eventually gets rejected. The only difference between the previous behavior and the new behavior would be that the `e` param would now contain the raw error message returned by the server like in the example of 401, it would be 
```
{
	status: 'error',
	message: 'You must be logged in to do this.',
}
```

Previously it was an instance of `Response` object from the browser 

https://github.com/RocketChat/Rocket.Chat/blob/57b12f3ba07c36383f9e53635744afe32e924904/apps/meteor/app/utils/client/lib/RestApiClient.ts#L46